util.js

```diff
@@ -52,3 +52,36 @@ exports.getTestModifiers = function getTestModifiers(node) {
 
 	return [];
 };
+
+const getMembers = node => {
+	const name = node.property.name;
+
+	if (node.object.type === 'MemberExpression') {
+		return getMembers(node.object).concat(name);
+	}
+
+	return [name];
+};
+
+exports.getMembers = getMembers;
+
+const assertionMethodsNbArguments = {
```

Comment:
They're more proper for data, better API, faster, and can handle any kind of key name.